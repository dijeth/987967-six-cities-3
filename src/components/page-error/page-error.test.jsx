import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PageError from './page-error.jsx';

Enzyme.configure({
  adapter: new Adapter()
});

it(`<PageError /> should be rendered correctly`, () => {
  const tree = Enzyme.mount(<PageError message="Error" onClose={()=>{}} />);

  expect(tree.getDOMNode()).toMatchSnapshot();
})
