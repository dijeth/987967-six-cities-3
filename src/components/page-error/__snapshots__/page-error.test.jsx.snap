// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageError /> should be rendered correctly 1`] = `
<div
  class="page page--gray page--login"
  style="position: fixed; top: 0px; width: 100vw; z-index: 1000;"
>
  <div
    class="page__main page__main--login"
  >
    <div
      class="page__login-container container"
    >
      <section
        class="login"
      >
        <h1
          class="login__title"
        >
          Error
        </h1>
        <button
          class="form__submit button"
          type="button"
        >
          Продолжить
        </button>
      </section>
    </div>
  </div>
</div>
`;
